docs/manual/install/index.rst

```diff
@@ -116,6 +116,13 @@ With the dependencies in place, you can now install the stable version of qtile
 
    pip install qtile
 
+Or with sets of dependencies:
+
+.. code-block:: bash
+
+   pip install qtile[wayland]  # for Wayland dependencies
+   pip install qtile[widgets]  # for all widget dependencies
+   pip install qtile[all]      # for all user dependencies
```

Comment:
Great idea!

From an outside view maybe "user" dependencies is a bit vague. Is this documented somewhere?

Reply:
If it's not documented a small sentence somewhere above or in the comment itself will do

Reply:
Good point, maybe just "all dependencies"?

Reply:
Oh that is much less confusing